source/guides/tooling/typescript-support.md

```diff
@@ -43,6 +43,16 @@ The `"types"` will tell the TypeScript compiler to only include type definitions
 You can find an example of Jest and Cypress installed in the same project using a separate `tsconfig.json` file in the {% url cypress-io/cypress-and-jest-typescript-example https://github.com/cypress-io/cypress-and-jest-typescript-example %} repo.
 {% endnote %}
 
+{% note warning %}
+You may have to restart your IDE's TypeScript server if the setup above does not appear to work. For example:
+
+VS Code (within a .ts file):
```

Comment:
I think it would be better to mention that restart TS server can help with intellisense while in a `.ts` or `.js` file

Reply:
@bahmutov Thanks for the review, I'll add js as well.